.eslintignore

```diff
@@ -0,0 +1 @@
+test/
```

Comment:
if you run 'npm run coverage' - it will create a coverage folder that's ignored in git and npm, but if you subsequently npm run lint it will try to verify the coverage folder and will fail miserably.

Reply:
Added